cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -653,13 +649,38 @@ public void requestLocationPermission() {
         ActivityCompat.requestPermissions(getCurrentActivity(), new String[]{Manifest.permission.ACCESS_FINE_LOCATION}, locationPermissionRequestCode);
     }
 
+    /**
+     * request for push notification
+     */
+    public void requestPushNotificationPermission() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            if (this.hasNotificationPermission()) {
+                return;
+            }
+        }
+        if (getCurrentActivity() == null) {
+            this.pendingRequestPushNotificationPermissionCall = true;
+            return;
+        }
+        this.pendingRequestPushNotificationPermissionCall = false;
+        ActivityCompat.requestPermissions(getCurrentActivity(), new String[]{Manifest.permission.POST_NOTIFICATIONS}, pushNotificationPermissionRequestCode);
+    }
+
     /**
      * to check if app has location permission
      */
     public boolean hasLocationPermission() {
         return ContextCompat.checkSelfPermission(getContext(), Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED;
     }
 
+    /**
+     * to check if app has notification permission
+     */
+    public boolean hasNotificationPermission() {
```

Comment:
Can you check the difference to this method:
https://github.com/cleverpush/cleverpush-android-sdk/blob/7b0726c4e5f280c5df21ecdce909533524a4a84e/cleverpush/src/main/java/com/cleverpush/CleverPush.java#L2558
Is "areNotificationsEnabled" still working for Android 13?